content/applications/productivity/mail_plugins/outlook.rst

```diff
@@ -75,6 +75,25 @@ Connect your database
       Only a limited amount of *Company Insights* (*Lead Enrichment*) requests are available as a
       trial. This feature requires :ref:`prepaid credits <mail_plugins/pricing>`.
 
+   .. warning::
+      If, after a short while, the panel is still empty. It is possible that your browser cookies
+      settings policy did prevent it to load.
+      Note that this policy does also change if you are connected in "Incognito" mode on your
+      browser.
+
+      To fix this issue, you can configure your browser to always allow cookies on Odoo's plugin page.
+
+      For Chrome in particular you can do so by following the guide at:
+      `https://support.google.com/chrome/answer/95647 <https://support.google.com/chrome/answer/95647#:~:text=Allow%20or%20block%20cookies%20for%20a%20specific%20site>`_
+      then add `download.odoo.com` in the list of `Sites that can always use cookies`.
```

Comment:
Small changes to keep the grammar consistent throughout the sentence.

`and adding download.odoo.com to the list of Sites that can always use cookies.`

Please note that I removed the backtick punctuation around "download.odoo.com" and "Sites that can always use cookies" in the corrected quote, because Github reads it as formatting.